packages/styles/scss/components/masthead/_masthead-search.scss

```diff
@@ -122,7 +122,7 @@
 
       @include carbon--breakpoint-down(md) {
         width: 100vw;
-        right: (-$carbon--spacing-09);
```

Comment:
Percy seems to show a visual difference that may have been caused by this change. Need investigation.

Reply:
![screencapture-localhost-3000-testpages-v18-base-rtl-html-2020-08-28-00_16_45](https://user-images.githubusercontent.com/1641214/91562210-d1619700-e90a-11ea-8c07-d80ed25c5db7.png)

Yes, this looks bad, why was this removed?

Reply:
Hmm....

Here's what was going on.  That `right` property was originally there to strategically place the search bar in the right spot over that user button while is actually lived along side it.  When I was originally developing this feature, when I removed the user button when the search bar is open on the medium breakpoint, the search bar would be placed too far to the right.  So the `right` property was removed to get the search bar back in the correct spot.

The question here is on the right side of the comparison, why is the user button not removed while the search bar is open.  Taking a look.

Reply:
I haven't been able to replicate this Percy view locally.  Is there a way I can get more info about how Percy generated this view?  Does Percy do a `click` event to open the search?  Or is there a way to directly load a page with the search already open?  ...maybe a page that Percy opens that has the `activate-search` property on the masthead container set to `true`?  Then the user button wouldn't get the click event that the search opened and set itself to be hidden.

Reply:
Probably a more important aspect of this is; The screenshot is of React. Meaning that the broken UI is not of Web Components but is of React.

Reply:
Ah.  I understand. The style is shared across the packages. Ok.